src/main/kotlin/lotto/output/ConsoleOutput.kt

```diff
@@ -0,0 +1,28 @@
+package lotto.output
+
+import lotto.domain.Score
+import lotto.domain.number.Numbers
+
+class ConsoleOutput {
+
+    fun purchaseResponse(tickets: List<Numbers>) {
+        val primitiveNumbers =
+            tickets.map { guessingNumber -> guessingNumber.winningNumbers.map { number -> number.number } }
+        println("bought ${primitiveNumbers.size} of them.")
+        primitiveNumbers.map { it -> it.sorted() }
+            .forEach { println(it) }
+    }
```

Comment:
I thought this is complicated for me to read, how about this?

```suggestion
    fun purchaseResponse(tickets: List<Numbers>) {
        println("bought ${tickets.size} of them.")

        tickets.forEach { ticket ->
            ticket.winningNumbers
                .map { it.number }
                .sorted()
                .also { println(it) }
        }
    }
```